src/modules/tests/test-data/cat-a-mod1/single-fault-competencies/single-fault-competencies.selector.ts

```diff
@@ -0,0 +1,28 @@
+import { SingleFaultCompetencies, TestData } from '@dvsa/mes-test-schema/categories/AM1';
+import { SingleFaultCompetencyNames } from '../../test-data.constants';
+import { CompetencyOutcome } from '../../../../../shared/models/competency-outcome';
+
+export const getSingleFaultCompetencies = (testData: TestData): SingleFaultCompetencies => {
+  return testData.singleFaultCompetencies;
+};
+
+export const hasCompetencyDrivingFault = (
+  singleFaultCompetencies: SingleFaultCompetencies,
+  competencyName: SingleFaultCompetencyNames,
+): boolean => {
+  return singleFaultCompetencies[competencyName] === CompetencyOutcome.DF;
+};
+
+export const hasCompetencySeriousFault = (
+  singleFaultCompetencies: SingleFaultCompetencies,
+  competencyName: SingleFaultCompetencyNames,
+): boolean => {
+  return singleFaultCompetencies[competencyName] === CompetencyOutcome.S;
+};
+
+export const hasCompetencyDangerousFault = (
+  singleFaultCompetencies: SingleFaultCompetencies,
+  competencyName: SingleFaultCompetencyNames,
+): boolean => {
+  return singleFaultCompetencies[competencyName] === CompetencyOutcome.D;
+};
```

Comment:
I would prefer to see these functions in separate helper files or a provider, particularly if they can be shared across categories, so we can avoid duplication.

Reply:
`SingleFaultCompetencies` comes from AM1 schema. might be difficult to reuse exactly these selectors